docs/elasticsearch-spec.asciidoc

```diff
@@ -171,16 +171,37 @@ spec:
 === Virtual memory
 
 By default, Elasticsearch uses memory mapping (`mmap`) to efficiently access indices.
-Usually, default values for virtual address space on Linux distributions are too low for Elasticsearch to work properly, which may result in out-of-memory exceptions.
-To increase virtual memory, ECK sets the recommended value by default.
+Usually, default values for virtual address space on Linux distributions are too low for Elasticsearch to work properly, which may result in out-of-memory exceptions. This is why the quickstart example disables `mmap` via the `node.store.allow_mmap: false` setting. For production workloads, it is strongly recommended to increase the kernel setting `vm.max_map_count` to `2621441` and leave `node.store.allow_mmap` unset.
```

Comment:
Do you want to link back to the 'quickstart' here, while it is abundantly clear for people working on ECK what is meant,  it might not be clear for other readers.